core/src/main/kotlin/org/evomaster/core/output/service/TestSuiteWriter.kt

```diff
@@ -273,33 +272,34 @@ class TestSuiteWriter {
     }
 
     private fun classDescriptionComment(solution: Solution<*>, lines: Lines, timestamp: String = "") {
-        lines.add("/**")
-        lines.add(
-            " * This file was automatically generated by EvoMaster on ${
+        val format = config.outputFormat
+        lines.startCommentBlock(format)
+        lines.addBlockCommentLine(
+            " This file was automatically generated by EvoMaster on ${
                 escapeDocs(
                     ZonedDateTime.now().toString()
                 )
-            }"
+            }", format
         )
         classDescriptionEmptyLine(lines)
 
         if (timestamp != "") {
-            lines.add(" ************************************ WARNING ************************************ ")
-            lines.add(" * This is an snapshot of the generated tests after $timestamp seconds elapsed. *")
-            lines.add(" * The execution of Evomaster has not finished. *")
-            lines.add(" ********************************************************************************* ")
+            lines.addBlockCommentLine(" ************************************ WARNING ************************************ ", format)
+            lines.addBlockCommentLine(" * This is an snapshot of the generated tests after $timestamp seconds elapsed. *", format)
+            lines.addBlockCommentLine(" * The execution of Evomaster has not finished. *", format)
+            lines.addBlockCommentLine(" ********************************************************************************* ", format)
         }
 
-        lines.add(" * The generated test suite contains ${solution.individuals.size} tests")
+        lines.addBlockCommentLine(" The generated test suite contains ${solution.individuals.size} tests", format)
         classDescriptionEmptyLine(lines)
-        lines.add(" * Covered targets: ${solution.overall.coveredTargets()}")
+        lines.addBlockCommentLine(" Covered targets: ${solution.overall.coveredTargets()}", format)
         classDescriptionEmptyLine(lines)
-        lines.add(" * Used time: ${searchTimeController.getElapsedTime()}")
+        lines.addBlockCommentLine(" Used time: ${searchTimeController.getElapsedTime()}", format)
         classDescriptionEmptyLine(lines)
-        lines.add(" * Needed budget for current results: ${searchTimeController.neededBudget()}")
+        lines.addBlockCommentLine(" Needed budget for current results: ${searchTimeController.neededBudget()}", format)
         classDescriptionEmptyLine(lines)
-        lines.add(" * ${solution.termination.comment}")
-        lines.add(" */")
+        lines.addBlockCommentLine(" ${solution.termination.comment}", format)
+        lines.endCommentBlock(format)
```

Comment:
weird... GitHub cancelled my comment... fortunately it was just one :)

anyway, maybe best to move `format` directly in constructor of `Lines` so that it does not have to given as input everywhere (and anyway it would not change for a test)

Reply:
Good idea, I'll most likely close this PR and open one straight from the EM clone with the change included if it's ok for you.

I had originally tried to push a branch to EM but something happened, that's why I moved to a fork. I'll try again and let you know in case there's any issue.

Reply:
@Pgarrett sounds good. btw, `master` branch is protected, cannot push directly to it

Reply:
Yes, obviously, but I tried pushing my own and was unable to do so. I'll try again @arcuri82 . Thanks!